modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators.Tests/Godot.SourceGenerators.Tests.csproj

```diff
@@ -8,6 +8,8 @@
     <Nullable>enable</Nullable>
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
+
+    <RollForward>LatestMajor</RollForward>
```

Comment:
Why is this needed?

Reply:
If we don't specify a rollforward and target `net6` for this project, one couldn't build/run the test with only the `net7` and/or `net8` SDK installed.

Reply:
I think you mean the runtime, not the SDK. But considering how all our projects target `net6`, I'm not sure rolling forward would be that beneficial. I also don't really expect most contributors to run the tests manually, they'll probably be run mostly in CI which uses the .NET 6.0 SDK explicitly so :shrug:.

I was also thinking about moving the tooling to .NET 8.0 sometime soon, so maybe we could just target `net8` in this project already?

Reply:
Runtime*, yes. I can drop this commit if needed. I needed to be able to test because I removed net6 from my system, but I can leave this unstaged. Or moving to `net8` is a solution, but it'll mean the CI needs to run `net8` also.

Reply:
On second thought, let's avoid moving to `net8` in this PR. I was thinking it'd be fine since this PR doesn't run the tests in CI but you are right, it would require CI to run `net8` and I don't want that decision to delay this.

So, about `<RollForward>`, it makes me a bit hesitant because of possible breaking changes in the .NET runtime but then again this is just unit tests anyway so maybe it's fine.

Reply:
I removed the rollforward in my last rebase. I just run with `DOTNET_ROLL_FORWARD` in my env locally 😉